rector.php

```diff
@@ -112,6 +112,6 @@
     ]);
 
     $parameters->set(Option::PHPSTAN_FOR_RECTOR_PATH, __DIR__ . '/phpstan-for-rector.neon');
-    $parameters->set(Option::PHP_VERSION_FEATURES, PhpVersion::PHP_74);
+    $parameters->set(Option::PHP_VERSION_FEATURES, PhpVersion::PHP_80);
     $parameters->set(Option::ENABLE_CACHE, true);
```

Comment:
:+1: I think we can remove this as `composer.json` php versoin should be used by default

Reply:
remove `PHP_VERSION_FEATURES` constant?

Reply:
I removed the `$parameters->set(Option::PHP_VERSION_FEATURES, PhpVersion::PHP_80);` https://github.com/rectorphp/rector-src/pull/159/commits/16cc001039be4f36c52bdae789e65f2d73624ac7